ext/standard/tests/mail/mail_include.inc

```diff
@@ -1,8 +1,8 @@
 <?php
 // Change these to make tests run successfully
-$server   = '{localhost}';
+$server   = '{127.0.0.1:143}';
 $default_mailbox = $server . "INBOX";
-$domain = "example.com";
+$domain = "something.com";
```

Comment:
Is this needed to get the test working? As ``example.com`` is an IANA reserved domain (https://www.iana.org/domains/reserved)

Reply:
This is taken from the default IMAP configuration:
https://github.com/php/php-src/blob/b51a5c353ad7e1d702cba96b94517762821235b6/ext/imap/tests/setup/imap_include.inc#L7

I'm not against using `example.com` there.

Reply:
Changed that now.